doc/rtd/explanation/format.rst

```diff
@@ -50,22 +50,6 @@ archive.
 User data scripts can optionally render cloud instance metadata variables using
 jinja templating. See :ref:`instance_metadata` for more information.
 
-Example script
```

Comment:
Isn't this still a valid user data script? Why should we remove that part?

Reply:
Is there value to showing an arbitrary shell script?

Reply:
Not a ton, but I think there is some value. Without an example for each type, our format page is a dense wall of text that is harder to skim quickly.

I'd personally love to see each of our documented "User data format" types show - or link to in the case of gzip (if gzip is even a user data format) - a very simple example of a working user data of that format which could be passed to cloud-init, with an explanation of what it does.

Note: this section is still missing important information (do user-data scripts run on every boot? first boot? etc...)

Reply:
If you still want to remove it, I'm fine with that. It really doesn't add much on its own. That said, I'll probably add an example back in when implementing [this docs change](https://github.com/canonical/cloud-init/issues/4739) at some point in the future.